src/app/projects/project-details/project-details.component.html

```diff
@@ -4,9 +4,8 @@
       <div class="top">
         <span class="breadcrumbs"><a routerLink="/">Projects &gt;</a>
           <h4>{{project.name}}</h4>
-          <i [ngClass]="(localStorageProjectsService.projectIsFavorite(project.projectId))
-            ? 'pi pi-star favorite-project-icon' : 'pi pi-star-o favorite-project-icon'">
-          </i>
+          <i class="icon" (click)="onFavoriteIconClicked($event)" [ngClass]="(localStorageProjectsService.projectIsFavorite(project.projectId))
+            ? 'pi pi-star favorite-project-icon' : 'pi pi-star-o favorite-project-icon'"></i>
```

Comment:
Clickable items should be `<a>`, `<button>`, `<input>`, or have `role="button"`.

Suggest changing to `<button>`

Reply:
Changed to `<button>` with  `<i>` inside.
Not sure if the way I solved the styling is good or not.